packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -844,4 +848,60 @@ describe('Table', () => {
             expect(table.validity.duplicateRecordId).toBeTrue();
         });
     });
+
+    describe('with validatable columns', () => {
+        let element: Table<SimpleTableRecord>;
+        let connect: () => Promise<void>;
+        let disconnect: () => Promise<void>;
+        let column1: TableColumnValidationTest;
+
+        // prettier-ignore
+        async function setupWithTestColumns(): Promise<Fixture<Table<SimpleTableRecord>>> {
+            return fixture<Table<SimpleTableRecord>>(
+                html`<nimble-table>
+                    <${tableColumnValidationTestTag} foo bar id="first-column" field-name="stringData">Col 1</${tableColumnValidationTestTag}>
+                    <${tableColumnValidationTestTag} foo bar id="second-column" field-name="moreStringData">Col 2</${tableColumnValidationTestTag}>
+                </nimble-table>`
+            );
+        }
+
+        beforeEach(async () => {
+            ({ element, connect, disconnect } = await setupWithTestColumns());
+            column1 = element.querySelector<TableColumnValidationTest>(
+                '#first-column'
+            )!;
+        });
+
+        afterEach(async () => {
+            await disconnect();
+        });
+
+        it('updates invalidColumnConfiguration and validity when column state changes', async () => {
+            await connect();
```

Comment:
Could move the await connects to the beforeEach.

Reply:
Done